Chinese/learnbranch/learnbranch.md

```diff
@@ -0,0 +1 @@
+One day, Wukong found something called git branch. He feels that this is similar as his capability of split himself into several copies. He is very interested and starts learn the command.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.